bin/gaffer.cmd

```diff
@@ -19,6 +19,9 @@ call :prependToPath "%GAFFER_ROOT%\ops" IECORE_OP_PATHS
 
 call :prependToPath "%GAFFER_ROOT%\resources\IECoreUSD" PXR_PLUGINPATH_NAME
 call :prependToPath "%GAFFER_ROOT%\materialX" PXR_MTLX_STDLIB_SEARCH_PATHS
+if "%PXR_USD_WINDOWS_DLL_PATH%" EQU "" (
+	set PXR_USD_WINDOWS_DLL_PATH=%GAFFER_ROOT%/lib
```

Comment:
One question about this - we already call `add_dll_directory()` for `GAFFER_ROOT/lib`, right? So is this about stopping USD using `PATH` more than it is about getting USD to use `GAFFER_ROOT/lib`? If so, would it be more accurate to set the env -var to `""`?

Reply:
Yes, I guess this is more about preventing `PATH` entries from getting used since we're already setting the right binary path. I amended the commit and the commit wording and changelog entry to reflect that.

I also verified this works by pointing my `PATH` at a conflicting USD installation, checking that it fails without this patch and that the patch fixes it.